.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+defaults: &defaults
+  working_directory: /home/circleci/grafana-kentik-app
+  docker:
+    - image: circleci/node:10.9.0-jessie
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      #- run:
+      #    name: Update npm
+      #    command: 'sudo npm install -g npm@6.4.0'
+      - restore_cache:
+          keys:
+            - v2-dependencies-{{ checksum "package.json" }}
```

Comment:
Since we use yarn to manage packages, we should check `yarn.lock` file, like:
```yml
- restore_cache:
    keys:
      - dependency-cache-{{ checksum "yarn.lock" }}
```